apps/webapp/src/components/staking/validators-table.tsx

```diff
@@ -1,72 +1,68 @@
-import {
-  Card,
-  CardContent,
-  CardHeader,
-  CardTitle,
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableHeader,
-  TableRow,
-} from '@penumbra-zone/ui';
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@penumbra-zone/ui';
 import { getValidator } from '@penumbra-zone/types';
-import { useValidatorInfos } from './use-validator-infos';
 import { Oval } from 'react-loader-spinner';
 import { ValidatorInfoRow } from './validator-info-row';
+import { ValidatorInfoContext } from './validator-info-context';
+import { ValidatorInfo } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/component/stake/v1/stake_pb';
+import { ReactNode } from 'react';
 
-const HEADERS = ['Validator', 'Voting power', 'State', 'Bonding state', 'Commission'];
+const HEADERS = ['Validator', 'Voting power', 'Commission', 'Staking'];
 
-export const ValidatorsTable = () => {
-  const { loading, error, validatorInfos, votingPowerByValidatorInfo } = useValidatorInfos();
+interface ValidatorsTableProps extends ValidatorInfoContext {
+  /**
+   * Content to display inside the Staking cell.
+   */
+  renderStakingActions: (validatorInfo: ValidatorInfo) => ReactNode;
+}
 
+export const ValidatorsTable = ({
```

Comment:
Two main changes in this file:

1. Remove the `<Card>` wrapper, since that's now provided by either `<AllValidators />` or `<Account />`.
2. Accept props for the validator infos/etc., rather than grabbing them from the context. This is needed because the parent components may be filtering the validator infos (such as the `<Account />` component, which only renders the validator infos for a given account).